README.md

```diff
@@ -47,6 +49,8 @@ const App = () => <div>Hello World!</div>
 export default hot(module)(App)
 ```
 
+Do not use `hot` if you are using **parcel** bundler. It was designed for webpack.
```

Comment:
We should make it work on Parcel too.

Reply:
Can you explain what is the problem? I think we should open a new issue on Parcel, if it does not work with this project, it is useless.

Reply:
https://github.com/parcel-bundler/parcel/issues/619
Meanwhile, I've found a bug in `hot`. From now `hot` will throw an error is `.id` is not defined along the module.

Reply:
Follow up - https://github.com/gaearon/react-hot-loader/pull/818